### ISSUE TITLE
Make text "Login with OpenID Connect" customizeable via settings

### ISSUE THREAD

**Issue by Fregf** [oidc-wp/openid-connect-generic#476]
I would like an easy way to change the text "Login with OpenID Connect" on the button on wp-admin to something more specific for our users, such as: "Login via organization SSO", or something like that. There is currently no way to do this in the Settings of this plugin.

**Comment by Fregf** [comment]
Just found out that this plug-in adds this setting: https://github.com/timnolte/oidc-keycloak-sso

**Comment by timnolte** [comment]
@Fregf yes, so this plugin has a hook to do this, and there is a decent API setup that allows you to add your own settings into the plugin screen which is what that add-on I created uses. This is an aspect that I've had mixed feelings about. At the heart of this plugin there is a ton of customization ability for people to use if you're a developer. The plugin wasn't created as a non-developer first sort of plugin. However, your request is not the first so I've considered at the very least perhaps we provide some add-ons that would provide some of this functionality without the need for a developer.

**Comment by timnolte** [comment]
@Fregf I thought 8 had maybe already made an add-on specifically for this and I did. You can find an add-on here: https://github.com/oidc-wp/oidc-generic-button-text-addon

**Comment by rasos** [comment]
Ideally we would be able to even define an image such as key :key: for the button.

**Comment by timnolte** [comment]
> Ideally we would be able to even define an image such as key :key: for the button.  Just tested that the key emoji could already be just copied and used in the button field. 

If you want an icon like this it would be a CSS styling aspect that you can do already. Given the vast possibilities of custom login form designs and themes I don't expect there will be any dashboard element provided to supply an icon.

I'm still on the fence as to whether to move the text customization permanently into the settings when it can already be accomplished via a hook, and further added to the settings screen via a hook.
